### PR TITLE
added more content to get involved page

### PR DESCRIPTION
**Pull request by seankross**
## Description

See title

### CODE REVIEW

File: getinvolved.qmd [hunk 1]
```diff
@@ -5,15 +5,25 @@ toc: false
 
 ## DASL Guide
 
-The [WILDS Contributor Guide][guide] is the home for resources and best practices for WDL and Nextflow workflows, research compendia, Docker files and software.
+We maintain the [WILDS Contributor Guide][guide] as a collection of guidelines,
+review criteria, and best practices for developing production-ready WDL and 
+Nextflow workflows, research compendia, Docker files and software.
 
-In addition, the guide outlines the best practices that we strive to follow while developing WILDS.
+Our goal in developing the WILDS Contributor Guide is not just limitted to the
```
Comment (sckott) [inline getinvolved.qmd:12]: spelling on "limitted"

File: getinvolved.qmd [hunk 2]
```diff
@@ -5,15 +5,25 @@ toc: false
...
+a pull request in the appropriate GitHub repository. For example, if you want to 
+make a contribution to the [sixtyfour][] package, open a pull request in 
+the repository for that package at <https://github.com/getwilds/sixtyfour/>.
 
+If you are interested in making another kind of contribution, we would be happy
+to chat. Please contact us at `wilds@fredhutch.org`.
 
 [guide]: https://getwilds.org/guide/
-[sixtyfour]: https://github.com/getwilds/sixtyfour/
+[six]: https://github.com/getwilds/sixtyfour/
```
Comment (sckott) [inline getinvolved.qmd:29]: `[sixtyfour]`

**Review by sckott — CHANGES_REQUESTED** [review]
Should we say what kinds of contributions we're open to? e..g, 

- code
- docs
- tests
- ...

**Comment by seankross** [comment]
Thanks for the feedback @sckott, new updates are ready for you.

File: getinvolved.qmd [hunk 3]
```diff
@@ -5,15 +5,31 @@ toc: false
...
+the repository for that package at <https://github.com/getwilds/sixtyfour/>.
+We are interested in accepting many different types of contributions. If you
+have not contributed to WILDS in the past, get in touch by emailing us and we
+would be happy to discuss the contribution you have in mind. Contributions in
+the form of reporting reproducable bugs, bug fixes, new  features, improved 
+documentation, additional software tests, translation and localization, blogs,
+or tutorials are all welcome.
+
+If you are interested in making another kind of contribution, we would be happy
+to chat. Please contact us at `wilds@fredhutch.org`.
```
Comment (sckott) [inline getinvolved.qmd:32]: make this clickable so it opens up default email application? don't feel strongly just asking

**Review by sckott — COMMENTED** [review]
looks good just had one question about email link

File: getinvolved.qmd (hunk 3 above)
Comment (seankross) [reply getinvolved.qmd]: Good idea

**Comment by seankross** [comment]
ugh @sckott sorry for being annoying, just pushed again

**Review by sckott — APPROVED** [review]

**Comment by sckott** [comment]
I don't think the yml for the deploy previews in this repo has the old bot comment clean up, i'll add that

**Pull request merged**